Tests/DotnetBenchmark/Algebra.cs

```diff
@@ -27,6 +30,18 @@ public AlgebraTest()
         public void SolveHard() => exprHard.SolveNt(x);
         [Benchmark]
         public void SolveAnal() => exprSolvable.SolveEquation(x);
+        
+
+        private EDecimal dec = 3;
+        private EDecimal coef = EDecimal.FromDecimal(0.2m) ?? 0;
 
+        [Benchmark]
+        public void Cos()
+        {
+            dec = dec.Negate(MathS.Settings.DecimalPrecisionContext) ?? 0;
+            dec = dec.Multiply(dec, MathS.Settings.DecimalPrecisionContext) ?? 0;
+            dec = dec.Multiply(coef) ?? 0;
+            dec = dec.Cos(MathS.Settings.DecimalPrecisionContext) ?? 0;
```

Comment:
Why is the null-coalescing operator here? If these operations ever return null, we should throw early.

Reply:
They're never null, just didn't want to remove <WarningsAsErrors>

Reply:
Nullable reference types is only enabled for AngouriMath the project. Or you want them to be enabled everywhere? Can be done with a `Directory.Build.props`.